app/src/main/java/swen_anigans/mathematicfanatic/QuizResultsActivity.java

```diff
@@ -71,14 +71,8 @@ protected void onCreate(Bundle savedInstanceState) {
 
         Star[] starValues = getStars(correct,total,starGroup.getChildCount());
         setStars(starViews,starValues);
-        if(picked == null) {
-            setRewardListeners();
-        } else {
-            LinearLayout slideUp = (LinearLayout) findViewById(R.id.slideuppanel);
-            pickedTV = ((TextView) slideUp.getChildAt(0));
-            slideUp.removeAllViews();
-            formRewardSelectedLayout(slideUp,picked);
-        }
```

Comment:
Ohhh! I didn't think of that case. Hold on let me see and then fix if it doesn't work as intended.